<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AVR XML2SQLite</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="5">
     <widget class="QPushButton" name="pushButtonBrowseXML">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="6" column="7">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="4">
     <widget class="QLineEdit" name="lineEditXMLDir"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>XML's dir:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="8">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="pushButtonBrowseSQLITE">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="4">
     <widget class="QLineEdit" name="lineEditSQLITE"/>
    </item>
    <item row="6" column="0" colspan="6">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>321</width>
        <height>24</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>SQLITE file:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Truncate</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="pushButtonGo">
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>textEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
